include/fastrtps/types/TypeObject.h

```diff
@@ -1255,12 +1353,15 @@ class CompleteAliasHeader
     RTPS_DllAPI void deserialize(eprosima::fastcdr::Cdr &cdr);
 
     RTPS_DllAPI bool operator==(const CompleteAliasHeader& other) const;
+
+//    RTPS_DllAPI bool consistent(const CompleteAliasHeader &x,
```

Comment:
Why are some `consistent` functions commented?

Reply:
Because they are unused but are related somehow with annotations and I'm not sure if we must apply consistency policies to annotations applied to a type.